tests/pr_test.go

```diff
@@ -55,6 +55,12 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 		Prefix:        prefix,
 		ResourceGroup: resourceGroup,
 		Region:        region,
+		// Ignore updates to CBR rule - see https://github.com/terraform-ibm-modules/terraform-ibm-cos/issues/201
+		IgnoreUpdates: testhelper.Exemptions{List: []string{
+			"module.cos_bucket1.module.instance_cbr_rule[0].ibm_cbr_rule.cbr_rule",
+			"module.cos_bucket1.module.bucket_cbr_rule[0].ibm_cbr_rule.cbr_rule",
+			"module.cos_bucket2.module.bucket_cbr_rule[0].ibm_cbr_rule.cbr_rule",
+		}},
```

Comment:
@vburckhardt Is this a one off ignore to get the upgrade test to pass? If so, lets just skip the upgrade for this PR, and remove this ignore code so it does remain for future PRs?

Reply:
Yes it's a once off. I think this is fine to leave as is for now.